src/booleans.js

```diff
@@ -1,61 +1,67 @@
 function negate(a) {
-  // your code here
+  return a === false;
 };
 
 function both(a, b) {
-  // your code here
+  return a === true && b === true;
 };
 
 function either(a, b) {
-  // your code here
+  return a === true || b === true;
 };
 
 function none(a, b) {
-  // your code here
-};
+  return a === false && b === false; 
+}
 
 function one(a, b) {
-  // your code here
+  return (a === true && b === false) || (a === false && b === true);
 };
 
 function truthiness(a) {
-  // your code here
+  if (!a) {
+    return false;
+  } else {return true;
+  }
```

Comment:
a shorter way is to pass the boolean value into the Boolean constructor return Boolean(a) or using the double NOT return !!a

  ! determines the "truth" of what a value is not:
The truth is that false is not true (that's why !false results in true)
The truth is that true is not false (that's why !true results in false)

!! determines the "truth" of what a value is not not:
The truth is that true is not not true (that's why !!true results in true)
The truth is that false is not not false (that's why !!false results in false)